RESTserver/src/main/java/com/softserve/if072/restservice/dao/Queries.java

```diff
@@ -0,0 +1,15 @@
+package com.softserve.if072.restservice.dao;
+
+/**
+ This class collects SQL queries.
+
+ @author Oleh Pochernin
+ */
+public final class Queries {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)